app/api/auth/routes.py

```diff
@@ -5,81 +5,112 @@
 
 from app import schemas
 from app.api import deps
-from app.core.security import hashing_password
+from app.core.config import settings
+from app.core.security import hash_password, create_token
+from app.crud.user import crud_user
 
 
 router = APIRouter()
 
 
-@router.post("/login/access-token", response_model=schemas.Token)
-def login_access_token(
+@router.post("/login", response_model=schemas.Token)
+def login(
     db: Session = Depends(deps.get_db), user_data: schemas.UserLogin = Body(...)
 ) -> Any:
     """
-    OAuth2 compatible token login, get an access token for future requests
+    Endpoint to allow users to login
+
+    :param db: DB session
+    :param user_data: Body object with email and password
+    :return: jwt access and refresh token
     """
-    ...
+    user = crud_user.authenticate(
+        db, email=user_data.email, password=user_data.password
+    )
+    if not user:
+        raise HTTPException(status_code=400, detail="Incorrect email or password")
+    elif not crud_user.is_active(user):
+        raise HTTPException(status_code=400, detail="Inactive user")
+    return {
+        "access_token": create_token(
+            {"user_id": user.id}, expire_minutes=settings.ACCESS_TOKEN_EXPIRE_MINUTES
+        ),
+        "refresh_token": create_token(
+            {"user_id": user.id}, expire_minutes=settings.REFRESH_TOKEN_EXPIRE_MINUTES
+        ),
```

Comment:
used several times {"user_id": user.id}, I think need create variable for "user_id": user.id" and using variable, becouse "user_id": user.id" - the same in this function